File: 0118-pascals-triangle/0118-pascals-triangle.cpp
class Solution {
public:
    vector<vector<int>> generate(int numRows) {
        vector<vector<int>> ans;
        for(int i=1;i<=numRows;i++){
            int a=1;
            vector<int> temp;
            temp.push_back(1);
            for(int j=1;j<i;j++){
                a=a*(i-j);
                a=a/j;
                temp.push_back(a);
            }
            ans.push_back(temp);
        }
        return ans;
    }
};

File: 0118-pascals-triangle/0118-pascals-triangle.py
class Solution:
    def generate(self, n: int) -> List[List[int]]:
        li=[]
        for i in range(1,n+1):
            ans=1
            lis=[]
            lis.append(1)
            for j in range(1,i):
                ans=ans*(i-j)
                ans=ans//j
                lis.append(ans)
            li.append(lis)
        return li